api/routes/notes/get-note.routes.js

```diff
@@ -13,6 +16,58 @@ router.get('/:noteId', canReadNote, async (req, res) => {
 
 	if (error) throw new Error(error.message);
 
+	return res.status(StatusCodes.OK).json({
+		username: req.params.username,
+		title: data[0].title,
+		content: data[0].current_content,
+		id: data[0].note_id,
+	});
+});
+
+// Create a note as a user
+router.post('/users/:username/notes', canCreateNote, async (req, res) => {
+	const { data, error } = await supabase.from('notes').insert([
+		{
+			author: req.user.user_id,
+			title: req.body.title,
+		},
+	]);
+
+	if (error) throw new Error(error.message);
+
+	return res.status(StatusCodes.CREATED).json({
+		id: data[0].note_id,
+		link: '/api/users/'
+			.concat(req.params.username)
+			.concat('/notes/')
+			.concat(data[0].note_id),
```

Comment:
You can use template literals instead of chaining calls to `String.prototype.concat`:
```javascript
link: `/api/users/${req.params.username}/notes/${data[0].note_id}`
```